cmd/telemetrygen/internal/common/config.go

```diff
@@ -124,18 +124,19 @@ func (c *Config) CommonFlags(fs *pflag.FlagSet) {
 
 	// custom headers
 	c.Headers = make(map[string]string)
-	fs.Var(&c.Headers, "otlp-header", "Custom header to be passed along with each OTLP request. The value is expected in the format key=\"value\"."+
-		"Note you may need to escape the quotes when using the tool from a cli."+
-		"Flag may be repeated to set multiple headers (e.g -otlp-header key1=value1 -otlp-header key2=value2)")
+	fs.Var(&c.Headers, "otlp-header", "Custom header to be passed along with each OTLP request. The value is expected in the format key=\"value\". "+
+		"Note you may need to escape the quotes when using the tool from a cli. "+
+		"Flag may be repeated to set multiple headers (e.g --otlp-header \"key1=\\\"value1\\\"\" --otlp-header \"key2=\\\"value2\\\"\")")
```

Comment:
Updated the PR
1. Further simplified the example by removing starting and trailing quotes, from `--otlp-header \"key1=\\\"value1\\\"\"` to `--otlp-header key1=\\\"value1\\\"`
2. Use backtick strings so that it becomes `--otlp-header key1=\"value1\"`